src/components/DpFlyout/DpFlyout.mdx

```diff
@@ -9,13 +9,24 @@ import DpFlyout from './DpFlyout'
 
```

Comment:
This file can be deleted, then.

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/395/commits/61ca453d3e9a5f124e56b27a2e23cf12bce93fb4